Sources/MapboxCoreNavigation/PassiveLocationDataSource.swift

```diff
@@ -192,13 +185,17 @@ public protocol PassiveLocationDataSourceDelegate: class {
 extension TileEndpointConfiguration {
     /**
      Initializes an object that configures a navigator to obtain routing tiles of the given version from an endpoint, using credentials that are consistent with the given directions service.
+     
+     - parameter tilesVersion: Routing tile version.
+     - parameter minimumDaysToPersistVersion: The minimum age in days that a tile version much reach before a new version can be requested from the tile endpoint.
+     - parameter removesOldLocalVersions: Whether to automatically remove older versions from the cache. By default, older versions are automatically removed.
      */
-    convenience init(directions: Directions, tilesVersion: String) {
+    convenience init(directions: Directions, tilesVersion: String?, minimumDaysToPersistVersion: Int?, removesOldLocalVersions: Bool = true) {
```

Comment:
I guess docs for `directions` parameter should be added as well.

Reply:
I couldn’t think of a good way to describe what the Directions object would be used for, other than for its credentials, so I replaced it with its credentials.